### ISSUE TITLE
[SR-8979] System library target documentation needs to warn about uniquely naming the umbrella header

### ISSUE THREAD

**Issue by gwynne** [apple/swift-package-manager#4755]

**Comment by heckj** [comment]
@gwynne I know this is an old issue - but I want to ask if this is still an issue - does an Umbrella header that you add with what amounts to a system library dependency still confuse Xcode, now that Xcode directly opens Package.swift and the generate-xcodeproj pieces have been deprecated?

I recently ported the legacy documentation - and the only place today that talks about module maps is in reference to using a system library as a dependency: https://docs.swift.org/swiftpm/documentation/packagemanagerdocs/addingsystemlibrarydependency

That said, I want to see better, deeper documentation so I'm doing the research to understand the various moving parts of a module map, and what's commonly done (or need to be done) when you're wrapping or interoperating with a C library from Swift. (issue #8628 covers adding a baseline article into Swift PM docs)

**Comment by heckj** [comment]
As I've been going through this, I think the issue doesn't present itself any longer - specific to an earlier version of Swift where you generated Xcode projects from Package.swift files. @gwynne if this isn't reality and there's something that's easy to go awry here, please comment here or poke me to re-open this. I'm working on more general module map documentation (and artifactBundle docs) now in the background, and would like to capture any relevant troubleshooting or tips details within that if possible.

**Comment by gwynne** [comment]
Yeah, this is ancient as sin, I'd completely forgotten this ever existed 😅. Definitely not still a thing.
